src/core/bpf/bind-iface/bind-iface.bpf.c

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+/* <linux/bpf.h> must precede <bpf/bpf_helpers.h> due to integer types
+ * in bpf helpers signatures.
+ */
+#include <linux/bpf.h>
+#include <bpf/bpf_helpers.h>
+
+/* VRF interface index to bind sockets to, set from userspace */
+const volatile __u32 ifindex = 0;
+
+SEC("cgroup/sock_create")
+int sd_bind_interface(struct bpf_sock *ctx)
+{
```

Comment:
hmm, we are a tiny bit inconsistent on this when it comes to our bpf code. but please place the `{` on the same line as the func name, i.e. remove one newline here